examples/GettingFeaturesFromDatabase/Database/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using GettingFeaturesFromDatabase.Database.Services;
+using Microsoft.EntityFrameworkCore;
+
+namespace GettingFeaturesFromDatabase.Database;
+
+public static class ServiceCollectionExtensions
+{
+    public static void AddDatabase(this IServiceCollection services)
+    {
+        services.AddDbContext<SqliteDbContext>(options =>
+        {
+            options.UseSqlite("Data Source=example.db");
```

Comment:
What's example.db here?

Reply:
It's the SQLite database file.

Reply:
Thanks. Now I see the application will create an example.db during initialization.